.github/workflows/bump-version/action.yml

```diff
@@ -1,15 +1,10 @@
-name: Bump Version
-description: Update package.json with the new version number & Sets the version as ENV variable.
+name: Get Package Version
+description: Get the version of the package.json file, and set it as an environment variable.
 
 inputs:
   CHANNEL:
     required: true
 
-outputs:
```

Comment:
Move this back in

Reply:
Add a "TODO" comment to remove this code

Reply:
Done